test/plugins/test-trace-http.ts

```diff
@@ -45,6 +45,9 @@ describe('test-trace-http', function() {
 
   afterEach(function() {
     common.cleanTraces();
+  });
+
+  after(function() {
```

Comment:
Looking at the CI, it seems like the creation of the server should be in a `beforeEach` rather than the `server.close` call be moved to `after`.

Reply:
Yeah, I just noticed the failures and am looking. I'm wondering how tests worked when the server was started only once and closed after each test.

Reply:
I was mistaken. `server.close()` doesn't invalidate the server itself, but just stops accepting connections. So it should be done from `afterEach()`. Reverted that change.